test/ticky-ticky.eventlog.reference

```diff
@@ -0,0 +1,122 @@
+165: End of heap profile sample (size 8)
+166: Start of heap profile (biographical) sample (size 16)
+167: Time profile cost-centre stack (size variable)
+168: Start of a time profile (size 8)
+181: User binary message (size variable)
+200: Begin concurrent mark phase (size 0)
+201: End concurrent mark phase (size 4)
+202: Begin concurrent GC synchronisation (size 0)
+203: End concurrent GC synchronisation (size 0)
+204: Begin concurrent sweep (size 0)
+205: End concurrent sweep (size 0)
+206: Update remembered set flushed (size 2)
+207: Nonmoving heap census (size 13)
+210: Ticky-ticky entry counter definition (size variable)
+211: Ticky-ticky entry counter sample (size 32)
+
+Events:
+292000: created capset 0 of type CapsetOsProcess
+292000: created capset 1 of type CapsetClockDomain
+293000: created cap 0
+293000: assigned cap 0 to capset 0
+294000: assigned cap 0 to capset 1
+299000: capset 1: wall clock time 1607818763s 73220000ns (unix epoch)
+299000: capset 0: pid 21680
+305000: capset 0: parent pid 9239
+312000: capset 0: RTS version "GHC-9.1.0.20201210 rts_debug"
+315000: capset 0: args: ["./HelloWorld","+RTS","-lT"]
+3772000: heap stats for heap capset 0: generations 2, 0 bytes max heap size, 1048576 bytes alloc area size, 1048576 bytes mblock size, 4096 bytes block size
+3857000: task 0x7fd27f405e90 created on cap 0 with OS kernel thread 32996106
+3861000: cap 0: creating thread 1
+3871000: cap 0: running thread 1
+4051000: cap 0: stopping thread 1 (thread finished)
+4053000: task 0x7fd27f405e90 deleted
+4055000: cap 0: creating thread 2
+4055000: task 0x7fd27f405e90 created on cap 0 with OS kernel thread 32996106
+4056000: cap 0: running thread 2
+4060000: cap 0: stopping thread 2 (thread finished)
+4061000: task 0x7fd27f405e90 deleted
+4084000: cap 0: starting GC
+4107000: cap 0: GC working
+4358000: cap 0: GC idle
+4359000: cap 0: GC done
+4363000: cap 0: GC idle
+4363000: cap 0: GC done
+4364000: cap 0: GC idle
+4364000: cap 0: GC done
+4405000: cap 0: allocated on heap capset 0: 51672 total bytes till now
+4406000: cap 0: finished GC
+4407000: cap 0: all caps stopped for GC
+4407000: cap 0: GC stats for heap capset 0: generation 1, 3320 bytes copied, 25256 bytes slop, 942080 bytes fragmentation, 1 par threads, 0 bytes max par copied, 3320 bytes total par copied, 0 bytes balanced par copied
+4407000: cap 0: live data in heap capset 0: 44376 bytes
+4408000: cap 0: size of heap capset 0: 2097152 bytes
+12419000: ticky counter sample 4535063528: entry count: 1, 0 allocs, 0 allocd
+12420000: ticky counter sample 4535063632: entry count: 1, 0 allocs, 0 allocd
+12421000: ticky counter sample 4535063728: entry count: 1, 0 allocs, 0 allocd
+12434000: ticky counter definition 4535063528, arity: 0, def kinds: , name: Main.main1{v r1sI} (fun)
+12434000: ticky counter definition 4535063632, arity: 0, def kinds: , name: Main.main2{v r1sJ} (fun)
+12434000: ticky counter definition 4535063728, arity: 0, def kinds: , name: :Main.main{v 01D} (fun)
```

Comment:
Is it expected that these ref kinds are empty?

Reply:
I believe so since those closures take no arguments (except go), but @bgamari would know better.

Reply:
Actually, this program has no `go` function. Sorry for the confusion. If you want, I can update the program to contain ticky events that have values there. Should I do that?

Reply:
Thanks! No I think it's fine as-is.